routes/api/v1/playlists.js

```diff
@@ -29,6 +29,40 @@ router.post('/', async (request, response) => {
   }
 });
 
+router.put('/:id', async (request, response) => {
+  let newTitle = request.body.title;
+
+  if (newTitle) {
+    let playlist = await database('playlists').where('id', request.params.id).first();
```

Comment:
The variable names here must have been hard to name. There are so many types of playlists!